README.md

```diff
@@ -1,56 +1,51 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# TO DO LISTS
 
+> A Web application that lets the user add, remove and reorder to do lists
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- Html,JS,CSS
+- Webpack and other dependencies
+- Git, Github, linters
 
-[Live Demo Link](https://livedemo.com)
+## Setup
 
+To get a local copy up and running you'll need to have [NodeJS](https://nodejs.org/en/download/) installed on your local machine.
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+### Install
 
+After installing NodeJS please follow the next steps...
 
-To get a local copy up and running follow these simple example steps.
+### Usage
 
-### Prerequisites
+```bash
 
-### Setup
+npm i
 
-### Install
+```
 
-### Usage
+-- for installing dev dependecies.
 
-### Run tests
+```bash
 
-### Deployment
+npm run build
 
+```
 
+-- to run the project.
 
-## Authors
+## Live Demo (Coming Soon)
```

Comment:
[OPTIONAL] Please include a link to a live demo in your readme file. It provided easy navigation and makes your project look professional. In addition, this is a webpack project. If you have any problems deploying the page, check this [link](https://gist.github.com/joshuapekera/ef364073b01fb0e21d3f)